popup.css

```diff
@@ -0,0 +1,489 @@
+.expandcard {
+  display: none;
+  background-color: rgb(193, 199, 208, 0.5);
+  font-family: "Poppins", sans-serif;
+  backdrop-filter: blur(5px);
+  position: fixed;
+  padding-top: 15px;
+  top: 0;
+  right: 0;
+  width: 100vw;
+  height: 100vh;
+}
+
+.container {
+  width: 100%;
+  margin: 114px auto;
+  display: grid;
+  gap: 88px;
+  font-family: "Poppins", sans-serif;
+}
+
+.card-body {
+  border-radius: 16px;
+  background-color: #fff;
+  padding: 16px;
+  width: 86%;
+  margin-top: 10%;
+  margin-bottom: 10%;
+  font-family: "Poppins", sans-serif;
+}
+
+.card-body:hover {
+  animation-name: card-body;
+  animation-duration: 3s;
+}
+
+@keyframes card-body {
+  from {
+    background-color: #e5e9f2;
+  }
+
+  to {
+    background-color: #fff;
+  }
+}
+
+/* Popup Styling start */
+.modal-body {
+  position: relative;
+  width: 320px;
+  height: 90%;
+  margin: 0 auto;
+  padding: 16px;
+  background-color: #fff;
+  border-radius: 16px;
+  z-index: 5;
+}
+
+#modal-close {
+  position: absolute;
+  cursor: pointer;
+  top: 30px;
+  right: 30px;
+}
+
+.line-break {
+  margin: 10px auto;
+  border: solid 1px #ebecf0;
+}
+
+.modal-button {
+  padding: 6px;
+  height: 40px;
+  width: 45%;
+  border-radius: 8px;
+  border: 1px solid #396df2;
+  color: #396df2;
+  background-color: #fff;
+  font-size: 12px;
+  font-weight: 500;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 5px;
+  transition: background-color 0.3s ease;
+}
+
+.modal-button:hover {
+  background-color: #e6e6e6;
+  text-decoration: underline;
+}
+
+.modal-combine {
+  display: flex;
+  align-items: center;
+}
+
+.modal-content > p {
+  font-size: 15px;
+  display: inline-block;
+}
+
+.svg {
+  margin-left: 5px;
+}
+
+.card-header {
```

Comment:
- Kindly reduce the bottom border here. 😃
- Your design
<img width="392" alt="image" src="https://user-images.githubusercontent.com/41163571/233776227-f8966136-5c3b-4e87-89d6-67f7352905eb.png">

- Figma design
<img width="197" alt="image" src="https://user-images.githubusercontent.com/41163571/233776264-7ac1c355-5948-420a-8dd6-fef8e056586b.png">